modules/operations/pages/astream-limits.adoc

```diff
@@ -65,6 +65,8 @@ Max throughput for a *non-partitioned topic* or a *single partition of a partiti
 |Not set
 |===
 
+*These guardrails cannot be changed.
+
```

Comment:
Note: Max Message Size can be changed for the dedicated plan.
We cannot change them on  the Serverless cluster